app/src/main/java/com/findmyrecycling/FindMyRecyclingApplication.kt

```diff
@@ -0,0 +1,22 @@
+package com.findmyrecycling
+
+import android.app.Application
+import org.koin.android.ext.koin.androidContext
+import org.koin.android.ext.koin.androidLogger
+import org.koin.core.context.GlobalContext
+import org.koin.core.logger.Level
+
+class FindMyRecyclingApplication {
+
+    class FindMyRecyclingApplication : Application() {
+        override fun onCreate() {
+            super.onCreate()
+            GlobalContext.startKoin{
+                androidLogger(if (BuildConfig.DEBUG) Level.ERROR else Level.NONE)
+                androidContext(this@FindMyRecyclingApplication)
+                modules(appModule)
+            }
+
+        }
+    }
+}
```

Comment:
Added FindMyRecylingApplication.kt file to link with the AppModule.kt file, this again is a file needed for Koin, reference for Application constructors was used from the following site: 
https://developer.android.com/reference/kotlin/android/app/Application